docs/source/conf.py

```diff
@@ -292,5 +296,57 @@ def inject_minigalleries(app, what, name, obj, options, lines):
         lines.append("\n")
 
 
+def inject_weight_metadata(app, what, name, obj, options, lines):
+
+    if obj.__name__.endswith("_Weights"):
+        lines[:] = ["The model builder above accepts the following values as the ``weights`` parameter:"]
+        lines.append("")
+        for field in obj:
+            lines += [f"**{str(field)}**:", ""]
+
+            table = []
+            for k, v in field.meta.items():
+                if k == "categories":
+                    continue
+                elif k == "recipe":
+                    v = f"`link <{v}>`__"
```

Comment:
I understand that this is in order to avoid printing the whole URL on the documentation and instead add a link. It doesn't seem to work [here](https://output.circle-artifacts.com/output/job/aaddb6cd-567c-4552-90fa-296878ff5ab1/artifacts/0/docs/models/generated/torchvision.models.resnet18.html#torchvision.models.resnet18) though. Is the specific rendering old?

Reply:
Yeah, I added that as a commit later and didn't update the original message in the PR. Latest build is here https://output.circle-artifacts.com/output/job/8009776e-b101-426d-8766-98bd34ecb855/artifacts/0/docs/models/generated/torchvision.models.resnet18.html#torchvision.models.resnet18